.github/workflows/CI.yml

```diff
@@ -10,8 +10,8 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.3'
           - '1.6'
+          - '1.7'
```

Comment:
Let's add `1.8-beta4` as well, that should become important soon.

Reply:
Sorry, that should be `1.8.0-rc4` instead.

Reply:
Done.